### PR TITLE
Small fixes in users and groups controlpanel

### PR DESCRIPTION
**Pull request by nileshgulia1**
In continuation: https://github.com/plone/volto/pull/608#issuecomment-535321032

### CODE REVIEW

**Comment by nileshgulia1** [comment]
@tisto Everytime I run `yarn i18n` on volto it create these unused strings which eventually fail the builds.
``` js
#: components/manage/Display/Display
+msgid "Display"
+msgstr ""
```
`components/mosaic/Layout/Layout`
`components/manage/Widgets/ArrayWidget`
 `components/manage/Widgets/TokenWidget`

File: theme/themes/pastanaga/elements/segment.overrides [hunk 1]
```diff
@@ -108,3 +108,7 @@
   border: 3px dashed #ccc;
   box-shadow: none;
 }
+
+.ui.clearing.segment.actions {
```
Comment (sneridagh) [inline theme/themes/pastanaga/elements/segment.overrides]: @nileshgulia1 we just can't assume that all the segments clearing actions will be all uppercase... please use the scoped `.users-control-panel` for it.

Comment (nileshgulia1) [reply theme/themes/pastanaga/elements/segment.overrides]: @sneridagh Oops sorry, I missed that! :)

**Comment by tisto** [comment]
@nileshgulia1 did you try to run a "make clean"?

**Comment by nileshgulia1** [comment]
@tisto Yes. I tried but , no luck.

**Comment by tisto** [comment]
@sneridagh looks ok to me now, except the changelog which we can fix after the merge.

**Pull request merged**